.kres.yaml

```diff
@@ -28,8 +28,6 @@ spec:
   customCommands:
     - apk add --no-cache --update
         bash
-        binutils-aarch64
```

Comment:
do we need a new errata, but since signed uki's are never reproducable, I guess it doesn't add any value

Reply:
hopefully not, they are functionally same (and probably they weren't byte-by-byte same due to external dependency)